DatadogRUM/Sources/RUMVitals/VitalInfoSampler.swift

```diff
@@ -41,12 +41,22 @@ internal final class VitalInfoSampler {
 
     private var timer: Timer?
 
+    private static var maximumFramesPerSecond: Double {
+        get {
+            #if swift(>=5.9) && os(visionOS)
+            return 120.0 // Hardcoded to enable VisionOS compilation
```

Comment:
/question Would it be possible to not report the refresh rate instead of using a hardcoded and possibly wrong frame rate?

Reply:
This is just `maximum` frames per second.

It's a non-official support only. I think that exploring the right solution here requires some more demand from customers.

WDYT?